src/Breadcrumb.js

```diff
@@ -0,0 +1,47 @@
+import React, { cloneElement } from 'react';
+import classNames from 'classnames';
+import BootstrapMixin from './BootstrapMixin';
+import ValidComponentChildren from './utils/ValidComponentChildren';
+
+const Breadcrumb = React.createClass({
+  mixins: [BootstrapMixin],
+
+  propTypes: {
+    /**
+     * CSS classes for the wrapper `ol` element
+     */
+    className: React.PropTypes.string,
+    /**
+     * HTML id for the wrapper `ol` element
+     */
+    id: React.PropTypes.string
```

Comment:
I notice that the `Nav.js` also defines `className` and `id` propTypes, and pass `{...this.props}` to child component. https://github.com/react-bootstrap/react-bootstrap/blob/master/src/Nav.js#L20-L30. Is there any reason why `Nav.js` define them in propTypes? Thanks.